nntrainer/graph/network_graph.cpp

```diff
@@ -80,21 +80,22 @@ void NetworkGraph::updateConnectionName(const std::string &from,
     auto &layer = LNODE(node_list[i])->getObject();
     if (istrequal(layer->getName(), to))
       continue;
-    for (unsigned int j = 0; j < layer->input_layers.size(); ++j) {
-      if (istrequal(layer->input_layers[j], from)) {
-        layer->input_layers[j] = to;
-      }
-    }
+    LNODE(node_list[i])->updateInputLayers(from, to);
+    // for (unsigned int j = 0; j < layer->input_layers.size(); ++j) {
```

Comment:
Unintended comment?

Reply:
this is already removed in a 2nd commit.